server/routes/forms.ts

```diff
@@ -32,5 +36,23 @@ export default function routes({ dataFilterService }: Services): Router {
     })
   })
 
+  post('/github-repo-requests', async (req, res): Promise<void> => {
```

Comment:
Here is where your `<form>` `action` will point to and this is where the data from the form comes into the app.

Reply:
So something like `<form method="POST" action="/forms/github-repo-requests">`

Reply:
When you hit submit (make sure it's a submit button and not a generic button) it will post to the URL in the action.